build-zfs-module/Containerfile

```diff
@@ -1,4 +1,6 @@
-ARG BUILDER_VERSION=36
+# Needs to be set to the Fedora version 
+# on CoreOS stable stream, as it is our base image.
+ARG BUILDER_VERSION=37
```

Comment:
We could likely pull this out at the same time we do the `kernel-query` btw.

Reply:
True, I found it tricky passing it to the FROM line. I will investigate a bit more see if I missed something.